lib/install/task.js

```diff
@@ -106,7 +110,9 @@ class AptTask extends InstallTask {
   }
 
   async instDeb() {
-    const instDir = path.join(this.codeUri, '.fun', 'root');
+    const instDir = path.join(this.target || path.join(this.cwd, '.fun', 'root'));
+
+    const prefix = `${this.target ? 'FUN_INSTALL_PREFIX=' + this.target + ' ' : ''}`;
```

Comment:
const prefix = this.target ？`FUN_INSTALL_PREFIX=${this.target}  ' : '';

Reply:
done